locations/spiders/urban_outfitters.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+from scrapy.spiders import SitemapSpider
+
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class UrbanOutfitters(SitemapSpider, StructuredDataSpider):
+    download_delay = 10
+    name = "urban_outfitters"
+    item_attributes = {"brand": "Urban Outfitters", "brand_wikidata": "Q3552193"}
+    allowed_domains = ["www.urbanoutfitters.com"]
+    sitemap_urls = ["https://www.urbanoutfitters.com/robots.txt"]
+    sitemap_rules = [("", "parse_sd")]
+    sitemap_follow = ["/sitemapindex.xml", "/store_sitemap.xml", "/stores/"]
```

Comment:
`sitemap_follow` defines the sitemaps which should be parsed. So `/store_sitemap.xml` is the valid rule, but because we're now pointing directly at it. It's unneeded.
```suggestion
```
`/stores/` is the rule for the page urls that should be parsed from the sitemap. So it could be `sitemap_rules = [("/stores/", "parse_sd")]`, however in this case it's not necessary because all the pages in that sitemap are stores.